src/workerd/jsg/fast-api.h

```diff
@@ -0,0 +1,131 @@
+#pragma once
+
+#include <workerd/jsg/web-idl.h>
+
+#include <v8-fast-api-calls.h>
+#include <v8-local-handle.h>
+#include <v8-value.h>
+
+#include <kj/common.h>
+
+namespace workerd::jsg {
+
+template <typename T>
+struct IsFastApiCompatibleWrappedType1 {
```

Comment:
Can we come up with a better name for this type? I'm not very keen on the "1" at the end.

Reply:
This type will eventually need to be merged with type-wrapper. I'll remove this before landing.